packages/structure/src/outline/outline.ts

```diff
@@ -5,6 +5,8 @@ import { RWRoute } from '../model/RWRoute'
 import { URL_file } from '../x/URL'
 import { Command_cli, Command_open, TreeItem2 } from '../x/vscode'
 
+const DOCS = 'https://redwoodjs.com/docs'
```

Comment:
I don't think it's likely that this will change, I think it's less error prone, and flexible, to just link to the entire URL in each "withDocs" section. Let me know what you think.

Reply:
agreed!